tests/fixtures/db.js

```diff
@@ -17,11 +30,24 @@ const populateTables = async () => {
   // delete all rows in table
   await User.destroy({ where: {} })
 
-  // insert a sample user
-  await User.create(userOne)
+  await Promise.all(
+    // loop thru sampleUsers
+    sampleUsers.map(async user => {
+      // cache the hashed passwords
+      const hashedPassword = await bcrypt.hash(user.password, 8)
+
+      // insert sample users to DB with the hashed passwords
+      await User.create({
+        name: user.name,
+        email: user.email,
+        password: hashedPassword
+      })
+    })
+  )
```

Comment:
Wow that is bloody brilliant awaiting a Promise.all wrapper on an async array.map! You must have been going crazy trying to figure out why all your tests were failing randomly and inconsistently.

Reply:
Yes, I know, thank you.